### PR TITLE
New package: uzbl-0.9.0

### LINKED ISSUE #2399: Package Request: Uzbl
http://www.uzbl.org/

Well... everyone else is requesting browsers....

### PR DESCRIPTION
**Pull request by ebfe**
Closes #2399

### CODE REVIEW

File: srcpkgs/uzbl/template [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+do_build() {
+	CC="$CC" LD="$LD" CPP="$CPP" CFLAGS="$CFLAGS" LDFLAGS="$LDFLAGS" \
+	PREFIX=/usr \
+	make ${makejobs}
+}
+
+do_install() {
+	make PREFIX=/usr DESTDIR=${DESTDIR} install
+}
```
Comment (Vaelatern) [inline srcpkgs/uzbl/template]: Is there a reason you don't use `gnu-makefile`?

Comment (ebfe) [inline srcpkgs/uzbl/template]: > Is there a reason you don't use `gnu-makefile`?

It doesn't work. Less patching necessary this way.

File: srcpkgs/uzbl/template [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Template file for 'uzbl'
+pkgname=uzbl
+version=0.9.0
+revision=1
+hostmakedepends="pkg-config python3.4 python3.4-setuptools"
+makedepends="libsoup-devel webkitgtk-devel glib-devel"
+depends="python3.4 python3.4-six"
```
Comment (Vaelatern) [inline srcpkgs/uzbl/template]: Really need both? `python3.4-six` pulls in in `python3.4`.

File: srcpkgs/uzbl/template [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Template file for 'uzbl'
+pkgname=uzbl
+version=0.9.0
+revision=1
+hostmakedepends="pkg-config python3.4 python3.4-setuptools"
```
Comment (Vaelatern) [inline srcpkgs/uzbl/template]: `pkg-config python3.4-setuptools`

File: srcpkgs/uzbl/template [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+makedepends="libsoup-devel webkitgtk-devel glib-devel"
```
Comment (Vaelatern) [inline srcpkgs/uzbl/template]: `webkitgtk-devel glib-devel`

**Comment by Vaelatern** [comment]
:+1:

**Pull request merged**